creativecommons/search
# Add Thingiverse Search as external source

## Fixes
Fixes #94 by @dullbenz

## Description
update the search Portal to include Thingiverse search functionality

## Technical details
update `index.php`, add Thingiverse element to page display, add redirection to Thingiverse site in both `index.php` and `search.js`

## Tests

- On the search portal page, enter the search string
- Check/uncheck the filters you want to add/remove
- Confirm your search results on the Thingiverse page that opens
- If the __"modify"__ filter was checked, all the results displayed should have some license allowing modification and reuse. This can be verified by clicking one of the search results and inspecting it's licenses

## Screenshots 
![image](https://user-images.githubusercontent.com/40190640/196004469-beae385d-4c1b-41e1-8cb8-5dc7cef3d176.png)

## Checklist
 
  index.md`).
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin


For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
